src/app.rs

```diff
@@ -290,51 +290,63 @@ fn validate_date_argument(arg: String) -> Result<(), String> {
 }
 
 pub fn validate_time_format(formatter: &str) -> Result<(), String> {
-    let mut chars = formatter.chars();
-    loop {
-        match chars.next() {
-            Some('%') => match chars.next() {
-                Some('.') => match chars.next() {
-                    Some('f') => (),
+    let str = formatter.to_string();
+    let vec: Vec<&str> = str.split('\n').collect();
+
+    if vec.len() > 2 {
+        return Err("invalid format, can only contain one newline separator".to_owned());
```

Comment:
```suggestion
        return Err("invalid date format, cannot contain more than two entries".to_owned());
```
We can probably make that the error is from date format as it can also come from the env variable `TIME_STYLE`.